marketplace_standard_app_api/models/message_broker.py

```diff
@@ -5,14 +5,12 @@
 
 class MessageBrokerRequestModel(BaseModel):
     endpoint: str = Field(None, description="API endpoint of the application")
-    body_base64: str = Field(None, description="A base64-encrypted body")
-    body: Optional[Any] = Field(None, description="The request message payload")
     query_params: Optional[Dict[str, str]] = Field(None, description="Query parameters")
     headers: Optional[Dict[str, str]] = Field(None, description="Request headers")
+    body: Optional[str] = Field(None, description="The request message body encoded in base64")
```

Comment:
Is the base64 encoding of the message body mandatory or could it be a generic text stream (e.g. a JSON-object) as well?

Reply:
In the code I always encode the body into base64 format to make sure I end up with a string (otherwise, I might get errors). We could keep it generic but that might be misleading.

Reply:
I'm just wondering whether we need to enforce this or whether this could be left to the application.

Reply:
If we examine the response body, we could say that it is up to the application to provide for the response body a string that could be base64 either encoded or not, but the message broker service would still need to know whether it  should decode the body string. This perhaps could be solved by using a Boolean flag, which doesn't necessarily make things more elegant. In the request body, it's even more complicated - how should the message broker service know whether the message should be encoded or not? It can examine the content-type header but that complicates things. Therefore it seems easier to say that base64 is always used.

Reply:
Well, actually I'm not convinced myself. I can try again without encoding and see what happens

Reply:
Conceptually we are just forwarding the request, correct? So, whether the body is encoded or not could be specified as part of the [content-encoding header](https://httpwg.org/specs/rfc7231.html#header.content-encoding), no?

Reply:
Exactly. Indeed it works. See new commit: https://github.com/materials-marketplace/python-sdk/pull/11/commits/1024810568523cc88bd715ac8fee225ae0f96613

Reply:
Base64 is not omitted from the field description.